vignettes/rhino-fluent.Rmd

```diff
@@ -0,0 +1,321 @@
+  moduleServer(id, function(input, output, session) {
+    # Generate message to be displayed in UI
+    output$message <- renderText("👋 This text has been generated using shiny.fluent package inside a Rhino project!")
+  })
+}
+```
+
+The **`box::use`** function is used to load the necessary packages along with their required functions for the app.
+In this case, the app requires the **`shiny`** package for the basic Shiny functionalities and the **`shiny.fluent`** package for the Fluent UI design system.
+
+The **`ui`** function creates the user interface of the app.
+It takes an **`id`** argument which is used to namespace the UI elements.
+Inside the function, the **`NS`** function is called to create the namespace for the UI elements.
+The **`fluentPage`** function which is from `shiny.fluent` package is used to create the layout of the app.
+It consists of a heading (**`tags$h1`**) and a text output (**`textOutput`**) element that will display a message generated by the server.
+Fluent UI is a design system developed by Microsoft that provides a modern, responsive, and accessible user interface for web applications.
+
+The **`server`** function defines the server logic of the app.
+It takes an **`id`** argument which is used to match the namespace with the UI elements created in the **`ui`** function.
+Inside the function, the **`moduleServer`** function is called to create a Shiny module.
+The **`renderText`** function is used to generate the text output message that will be displayed in the UI.
+
+The exported **`ui`** and **`server`** functions can be used to run the app with the **`shinyApp`** function.
+The app will display a heading and a message that says "👋 This text has been generated using shiny.fluent package inside a Rhino project!" in the web browser.
+
+Upon running the app with `shiny::runApp()`, you should see the following on your screen!
+
+```{r, echo=FALSE}
+knitr::include_graphics("images/rhino_fluent_1.png")
+```
+
+## Adding modules to the Rhino App
+
+Your application runs, but it doesn't have any meaningful functionality.
+Let's add something there!
+
+In Rhino, each application view is intended to live as a [Shiny module](https://shiny.rstudio.com/articles/modules.html) and use encapsulation provided by [`box` package](https://klmr.me/box/).
+
+We will now try to code a module that basically creates a checkbox and shows values once checked or unchecked.
+
+Rhino already created a good place for new modules, the `app/view` directory.
+So, create a file there, named `checkbox.R` and paste the following code there -
+
+``` r
+# app/view/checkbox.R
+
+box::use(
+  shiny[moduleServer, NS, tags, textOutput, renderText, markdown], # Load shiny package and specific functions
+  shiny.fluent[fluentPage, Checkbox.shinyInput]  # Load shiny.fluent package and specific functions
+)
+
+#' @export
+ui <- function(id) {
+  ns <- NS(id)  # Create namespace for the app
+  fluentPage(  # Create a Fluent UI page
```

Comment:
A simple `div` would be enough, you already have a `fluentPage` in the main module.